src/DeviceSelector.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {useState} from 'react';
+import Button from '@material-ui/core/Button';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Select from '@material-ui/core/Select';
+
+import { getLocalDevices } from './services/local-devices';
+
+
+interface DeviceSelectorProps {
+    onSubmit: { (selectedVideoDevice: string, selectedAudioDevice: string): void };
+    videoDevices?: MediaDeviceInfo[];
+    audioDevices?: MediaDeviceInfo[];
+}
+
+const DeviceSelector: React.FC<DeviceSelectorProps> = (props) => {
+    const [selectedCamera, setSelectedCamera] = useState<string>('Disabled');
+    const [selectedMic, setSelectedMic] = useState<string>('Disabled');
+
+    const handleCameraSelected = async (event: any) => {
+        let devices = await getLocalDevices()
+        console.log('Hi1')
```

Comment:
This might be an unintentional log?

Reply:
most definitely is 😆

Reply:
Also thank you for catching that. That was actually ended up being unused